pandas/core/indexing.py

```diff
@@ -2152,7 +2149,7 @@ def _convert_to_indexer(
             )
 
 
-class _ScalarAccessIndexer(_NDFrameIndexer):
+class _ScalarAccessIndexer(_NDFrameIndexerBase):
```

Comment:
The only _NDFrameIndexer method these use is _tuplify, which is why this PR takes that out and makes it a function instead of a method.  Much easier to reason about these classes now